exploratory_data_analysis/eda.py

```diff
@@ -5,6 +5,7 @@
 
 from utils.exploratory_data_analysis import get_eda_summary
 from utils.parsers import read_data
+from utils.plotting import plot_lines_by
```

Comment:
Please do not include this import as we are not using it. Pull from main to get the last version.